continuous_integration/environment-3.8-dev.yaml

```
@@ -5,8 +5,9 @@ channels:
 dependencies:
 - c-compiler
 - dask=2022.3.0
-- fastapi=0.69.0
+- fastapi=0.92.0
```

Think we want `>=` here to match what we have elsewhere in this PR

```suggestion
- fastapi>=0.92.0
```

---

The py 3.8 environment file generally tests tests against our minimum specified deps (Added in #555), so I think this might be intentional to test if things still work with 0.92 even if conda picks up a newer version.

---

Ah ok. Wasn't aware of that. Happy to revert